src/org/sipdroid/sipua/RegisterAgent.java

```diff
@@ -131,14 +132,16 @@
 	Message currentSubscribeMessage;
 	public final int SUBSCRIPTION_EXPIRES = 184000;
 
+	PublishAgent pas;

```

Comment:
I would be doing the final changes with variable name etc once @dpocock is done with testing. He is onto it right now.
Thanks for the suggestion :)